pinning.go

```diff
@@ -375,18 +402,29 @@ func (s *Server) handleListPins(e echo.Context, u *User) error {
 
 	}
 
+	var contents []Content
+	if err := q.Scan(&contents).Error; err != nil {
+		return err
+	}
+
+	contentIDs := make([]uint, len(contents))
```

Comment:
oh hey, we already have the slice of contents. Maybe we just make pinStatus accept a Content type? that way we dont have to go load it from the db again?